task/06-conditions-n-loops-tasks.js

```diff
@@ -119,7 +132,7 @@ function isTriangle(a,b,c) {
  *  
  */
 function doRectanglesOverlap(rect1, rect2) {
-    throw new Error('Not implemented');
+    return (rect1.height >= rect2.top && rect1.width >= rect2.left) ? true : false;
```

Comment:
ternary operator is redundant here
Please, see https://javascript.info/logical-operators